controllers/csi/provisioner/reconciler.go

```diff
@@ -220,8 +227,6 @@ func installAgent(rlog logr.Logger, dtc dtclient.Client, flavor, arch, targetDir
 }
 
 func unzip(rlog logr.Logger, r *zip.Reader, outDir string) error {
-	const agentConfPath = "agent/conf/"
-
 	os.MkdirAll(outDir, 0755)
```

Comment:
errors shouldn't be implicitly ignored, use `_ = os.MkdirAll(...)` to explicitly ignore them or handle them accordingly

Reply:
please double check all similar calls like `os.MkdirAll(...)` or `ioutil.WriteFile(...)`